### PR TITLE
Recording analytics for the Unincorporated epic.

### PR DESCRIPTION
**Pull request by abhiahirv1**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/views/common/what-is-your-role/what-is-your-role.njk [hunk 1]
```diff
@@ -30,21 +30,33 @@
...
+                      attributes: {
+                        "data-event-id": "I am a member of " + unincorporatedBusinessName
+                      }
                     } if (acspType == "UNINCORPORATED_ENTITY"),
 
                     {
                       value: "EQUIVALENT_OF_DIRECTOR",
                       text: i18n.equivalentToDirector + unincorporatedBusinessName
+                      attributes: {
+                        "data-event-id": "I am a director of " + unincorporatedBusinessName
```
Comment (ttingle-ch) [inline src/main/views/common/what-is-your-role/what-is-your-role.njk:50]: This should say "I am the equivalent of a director of "

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/views/common/what-is-your-role/what-is-your-role.njk [hunk 2]
```diff
@@ -30,21 +30,33 @@
...
+                      attributes: {
+                        "data-event-id": "I am a member of " + unincorporatedBusinessName
+                      }
                     } if (acspType == "PARTNERSHIP"),
 
                     {
                       value: "MEMBER_OF_GOVERNING_BODY",
                       text: i18n.memberOfGoverningBody + unincorporatedBusinessName
+                      attributes: {
+                        "data-event-id": "I am a member of " + unincorporatedBusinessName
```
[the remark this answers is not in the document]
Comment (abhiahirv1) [reply src/main/views/common/what-is-your-role/what-is-your-role.njk]: Done

File: src/main/views/common/what-is-your-role/what-is-your-role.njk (hunk 1 above)
Comment (abhiahirv1) [reply src/main/views/common/what-is-your-role/what-is-your-role.njk]: Done

**Review by ttingle-ch — APPROVED** [review]
LGTM

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by hemerem-ch — APPROVED** [review]

**Pull request merged**